### ISSUE TITLE
About CFG

### ISSUE THREAD

**Issue by hermanseu** [Lyy-iiis/imeanflow#4]
Hello, thank you very much for your great work!

Regarding formula 17 for CFG, I can't figure out how it was derived. Could you help me derive it?  @Gsunshine

**Comment by Morefre** [comment]
Hello, have you figured out the derivation? I also cannot understand how it is derived from the equation in the MeanFlow paper. Thank you in advance!

**Comment by JohnHerry** [comment]
Another question is about the two loss part, are they all required?    
In this project, the DiT model may produce both average-speed = U and aux-speed = V,  so that both fm-loss [Flow-Matching Loss ] and mf-loss[MeanFlow loss] can be computed.   
```
v_conditional_predicted = model(****,  t, r=t)
v_hat = (e0 - e1) * w + v_conditional_predicted * (1- w) 


 u_predicted,  dudt, v_aux = torch.func.jvp(model(***), (moel_args), (v_conditional_predicted, torch.ones_like(t), torch.zeros_like(t)))
v_expected = u_predicted + (t-r) * stopgrad(dudt)
mf_loss = ada_loss( v_expected - stopgrad(v_hat))
fm_loss = ada_loss(v_aux  - stopgrad(v_hat))

loss = mf_loss + fm_loss
```
In my task,  there is only one prediction output of DiT model,  so it can only predict the average-speed U [So no such v_aux in above code], so I had used only the mf_loss during training.   
```
loss = ada_loss(v_expected - v_hat)
```
but during traning, it seems that the iMeanFlow model is not  convergent， the loss is growing.   Is it becuase the `fm_loss` part is  indispensable in the model traning?   Can I use the `v_conditional_predicted` in replace of the `v_aux` to compute fm_loss in my case?
